apps/web-portal-client/lib/app/App.jsx

```diff
@@ -1,16 +1,74 @@
-import React from 'react'
+import React, {useState} from 'react'
 import {Layout} from '../components/layout/Layout'
 import {HeaderLayout} from '../components/layout/HeaderLayout'
 import {SidebarLayout} from '../components/layout/SidebarLayout'
 import {ContentLayout} from '../components/layout/ContentLayout'
+import DesktopMenu from '../components/layout/DesktopMenu.jsx'
+import MobileButtonMenu from '../components/layout/MobileButtonMenu.jsx'
+import Logo from '../components/layout/Logo.jsx'
+import MobileMenu from '../components/layout/MobileMenu.jsx'
 
 export function App() {
+  const [menuStatus, setMenuStatus] = useState(false)
+
   return (
-    <Layout
+    <div>
```

Comment:
The content below is OK, however, it was located not in the proper component, plz create `components/HomePage.jsx`  and add to the router with `/home` url